contracts/src/RedeemManager.1.sol

```diff
@@ -474,6 +476,10 @@ contract RedeemManagerV1 is Initializable, IRedeemManagerV1 {
             // we load the redeem request in memory
             params.redeemRequest = redeemRequests[_redeemRequestIds[idx]];
 
+            if (allowList.isDenied(params.redeemRequest.owner)) {
+                revert ClaimOwnerIsDenied();
+            }
+
```

Comment:
@iamsahu i feel like we might need to also check the params.redeemRequest.receipient against the DENY list ? As otherwise a blacklist wallet can still receive ETH, which doesn't seem like the right thing ?!

Reply:
The recipient of the redeem request is the same as the owner of the redeemRequest. So, we don't need a separate check.